### PR TITLE
Fix alt description for the images

### LINKED ISSUE #20735: Ensure no missing `alt` attributes (2)
### MDN URL

https://developer.mozilla.org/en-US/docs/

### What specific section or headline is this issue about?

All of MDN content

### What information was incorrect, unhelpful, or incomplete?

There used to be 404 missing `alt` attributes. We are down to 187. We need people to help complete the remaining 187 image descriptions.

### What did you expect to see?

Every image in MDNs content should have a useful `alt` attribute (the one exception:  code blocks explain bad HTML practices)

### Do you have any supporting links, references, or citations?

List of missing alt attributes: https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit#gid=0

### Do you have anything more you want to share?

Alt attributes aim to be short and concise, providing all the relevant information as the attribute value that the image conveys to a person who is able to see the image. 

When writing an `alt` attribute, think about the useful information the image conveys to you and how you would relay that information to someone who can read all of the page's content but just can't load images. What information does the image convey that is important and relevant to the user?

The content of the `alt` attribute for an image differs based on the context. For example, if the photo of Fluffy the dog is an avatar next to a review for Yuckymeat dog food, `alt="Fluffy"` is appropriate. If the photo is part of Fluffy's adoption page for the animal rescue society, the information conveyed in the image that is relevant for a prospective dog parent that is not duplicated in the surrounding text should be included. A longer description, such as `alt="Fluffy, a tri-color terrier with very short hair, with a tennis ball in her mouth."` is appropriate. As the surrounding text likely has Fluffy's size and breed, you would not included in the `alt`: that information is provided in the content and would therefore be redundant. As the dog's biography likely doesn't include hair length, colors, or toy preferences, which is important for adoption reasons but not in a dog food review. Don't describe the image: the prospective parent does not need to know if the dog is in- or out-doors, or that it has a red collar and a blue leash

All information image conveys that a sited user can access and is relevant to the context is what needs to be conveyed; nothing more. Keep it short, precise, and useful.

With screenshots, write what is learned from the image, don't detail the contents of the screenshot. You can omit information they don't need or already know. For example, if you're on a page about changing settings on Bing, if you have a screenshot of a Bing search result, don't include the search term or number of results, etc. as those are not the point of the image. Limit the alt to the topic at hand: how to change settings in Bing. The alt might be "The settings icon is in the navigation bar below the search field." Don't include "screenshot" or "Bing" as the user doesn't need to know it's a screenshot and already knows it's Bing as they are on a page explaining changing Bing settings.

### PR DESCRIPTION
**Pull request by ghost**
#20735

### CODE REVIEW

**Comment by Josh-Cena** [comment]
@najmiehsa Please don't use "fixes" if your PR doesn't _close_ an issue.

**Comment by najmiehsa** [comment]
@Josh-Cena ok thanks for the correction.

File: files/en-us/web/api/css_object_model/determining_the_dimensions_of_elements/index.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ There are several properties you can look at in order to determine the width and
 
 If you need to know the total amount of space an element occupies, including the width of the visible content, scrollbars (if any), padding, and border, you want to use the {{DOMxRef("HTMLElement.offsetWidth")}} and {{DOMxRef("HTMLElement.offsetHeight")}} properties. Most of the time these are the same as width and height of {{DOMxRef("Element.getBoundingClientRect()")}}, when there aren't any transforms applied to the element. In case of transforms, the `offsetWidth` and `offsetHeight` returns the element's layout width and height, while `getBoundingClientRect()` returns the rendering width and height. As an example, if the element has `width: 100px;` and `transform: scale(0.5);` the `getBoundingClientRect()` will return 50 as the width, while `offsetWidth` will return 100.
 
-![](dimensions-offset.png)
+![properties to determine the height and width of the element layout](dimensions-offset.png)
```
Comment (bsmth) [inline files/en-us/web/api/css_object_model/determining_the_dimensions_of_elements/index.md:25]:
```suggestion
![How the offsetWidth and offsetHeight properties are determined, considering padding, borders, and margin sizes](dimensions-offset.png)
```

File: files/en-us/web/api/css_object_model/determining_the_dimensions_of_elements/index.md [hunk 2]
```diff
@@ -22,13 +22,13 @@ There are several properties you can look at in order to determine the width and
...
 
 ## What's the size of the displayed content?
 
 If you need to know how much space the actual displayed content takes up, including padding but not including the border, margins, or scrollbars, you want to use the {{DOMxRef("Element.clientWidth")}} and {{DOMxRef("Element.clientHeight")}} properties:
 
-![](dimensions-client.png)
+![properties to determine the size of the content](dimensions-client.png)
```
Comment (bsmth) [inline files/en-us/web/api/css_object_model/determining_the_dimensions_of_elements/index.md:31]:
```suggestion
![How the clientWidth and clientHeight properties are determined, considering padding, borders, and margin sizes](dimensions-client.png)
```

File: files/en-us/web/api/css_object_model/determining_the_dimensions_of_elements/index.md (hunk 1 above)
Comment (najmiehsa) [reply files/en-us/web/api/css_object_model/determining_the_dimensions_of_elements/index.md:25]: the description should be long! thank you for the review.

**Review by bsmth — APPROVED** [review]
Thanks @najmiehsa 🙌🏻

**Comment by najmiehsa** [comment]
@bsmth I need to remove the changes to the last file from the PR.
I understood, the alt tags should be clear and descriptive. I make changes to the other one.
Thank you for the review.

**Comment by najmiehsa** [comment]
thanks!

**Pull request merged**